seclang/parser.go

```diff
@@ -142,6 +145,7 @@ func (p *Parser) evaluate(data string) error {
 	p.options.Config.Set("last_profile_line", p.currentLine)
 	p.options.Config.Set("parser_config_file", p.currentFile)
 	p.options.Config.Set("parser_config_dir", p.currentDir)
+	p.options.Config.Set("parser_root", p.root)
```

Comment:
This should replace the `working_dir` declared in https://github.com/corazawaf/coraza/pull/393/files#diff-f73e6b849d0fa747f9e706cbdb5ec76083f712530e21d64ea917cc37eb6347c5R153

Reply:
Sorry don't quite follow the link, do you mean remove `p.options.Config.Set/Get("working_dir")` and have `p.root` default to working dir if user doesn't call `SetRoot`?

In that case, this won't work even if the user doesn't call `SetRoot`
```
coraza/waf.exe
config/rules.conf
```

Since `..` wouldn't work. While we are discussing being OK with `../` not working with an explicit call to `SetRoot`, I'm not sure about when it hasn't been called at all, I guess it should be allowed though.